manifests/repo.pp

```diff
@@ -8,16 +8,13 @@
   Boolean $manage_container_manager          = $k8s::manage_container_manager,
   K8s::Container_runtimes $container_manager = $k8s::container_manager,
   String[1] $crio_version                    = $k8s::version.split('\.')[0, 2].join('.'),
+  Boolean $use_kubic_repos                   = false
```

Comment:
please document the new parameter with puppet-strings

Reply:
Happy to do all the tidy bits once I get consensus that this is an approach ya'all agree with.  If I can get pre-approval that going this direction is agreeable (dropping all obsolete k8s version support, etc)  I'm happy to invest a few hours making it perfect

Reply:
Personally not against dropping kubic entirely, optionally even with a catalog fail condition on Kubernetes <1.24 versions, since those versions are _long_ since deprecated.
Unfortunately the migration from kubic to the new isv:kubernetes repos can be difficult to automate, since some packages have migrated concerns between them.

On Debian, a complete uninstall of cri-o and conmon is necessary before the isv:kubernetes cri-o package can be installed - as one example.